docs/contribute/index.md

```diff
@@ -0,0 +1,9 @@
+# Getting Started
+
+> Become a part of OpenTrafficCam
+
+We are happy if you contribute your own code to OpenTrafficCam.
+This can be bug fixes in the current code, or even completely new stuff.
+It is best to start at [GitHub](github.md) to get a first overview.
+
```

Comment:
Add a section that even testing the framework and creating issues in the Github repositories or updating the [documentation](https://opentrafficcam.org/contribute/documentation/) is appreciated?

Reply:
This way?

Reply:
Maybe also issues with code, not only with documentation?

Reply:
It's all on the GitHub page. I think we don't need it twice.

Reply:
But whats with the OTC users who just download the framework without ever visiting the github page?

Reply:
They are not reporting any code issues ;)